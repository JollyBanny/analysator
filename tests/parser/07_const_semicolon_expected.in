const constant1 = 10;
      constant2 = 20
      constant3 = 30;
begin
end.