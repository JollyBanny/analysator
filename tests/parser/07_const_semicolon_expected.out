[Line: 3; Column: 15]	';' expected but 'constant3' found
